src/Transfers/ImportModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿//-------------------------------------------------------------
+// <copyright file="ImportModel.cs" company="GlobalLink Vasont">
+// Copyright (c) GlobalLink Vasont. All rights reserved.
+// </copyright>
+//-------------------------------------------------------------
+namespace Vasont.Inspire.Models.Transfers
```

Comment:
I believe this should be renamed to be more aligned with the scope of it's use. Something like ImportConfigurationModel